include/openssl/base.h

```diff
@@ -203,6 +203,11 @@ extern "C" {
 
 #define AWSLC_API_VERSION 16
 
+// In order to keep track of any changes to AWLC we introduce a string that tracks with the public Github
+// release version at https://github.com/awslabs/aws-lc
+
+#define AWSLC_VERSION_NUMBER_STRING "0.0.2"
```

Comment:
FYI:  The version string we sent over to NIST is 1.0.0.  Could you update the value?

Reply:
Talked offline, planning to do a quick follow up PR to update this string on the fips branch